src/test/java/spoon/test/generics/GenericsTest.java

```diff
@@ -1561,4 +1565,17 @@ public void testExecutableTypeParameter() {
 		assertEquals(formalType, ((CtTypeReference) m1.getActualTypeArguments().get(0)).getTypeParameterDeclaration());
 		assertNull(m1.getType().getTypeParameterDeclaration());
 	}
+
+	@org.junit.jupiter.api.Test
```

Comment:
Can't you convert the import to junit 5?

Reply:
I'd rather not touch the other tests